docs/NSR.md

```diff
@@ -347,11 +340,11 @@ The CA MUST enforce the use of Multi-Party Control for physical access to any Ro
 
 The CA SHOULD ensure passwords used as authentication credentials for accounts on CA Infrastructure, Network Equipment, or Workstations are generated and managed in accordance with NIST 800-63B Appendix A.
 
-The CA SHALL NOT require periodic password changes with a period less than two (2) years.
+The CA SHOULD NOT require periodic password changes with a period less than two (2) years.
```

Comment:
Are we making this change because Martijn is using some kind of system or service that has an outdated understanding of best practices and requires a password change at least every two years exactly?

Reply:
Raising a concern isn't the same as using a system ;)

However, it also seems like an appropriate alternative to the language that was removed in NSR 2.0, see https://groups.google.com/a/groups.cabforum.org/g/netsec/c/Ve1svWA-80Q/m/rZ6ozyp-AwAJ

Reply:
This was raised here particularly in the context of shared or service passwords, which it can be wise to rotate in periodic cycles shorter than 2 years. 
This requirement makes the most sense in cases where humans must remember the password, but following best practices and using credential managers makes this nearly meaningless as the human is not expected to remember passwords. Some systems of automated credential management rotate passwords frequently in the background and this would seem to disallow use of such features.